tangelo/algorithms/variational/iqcc_ilc_solver.py

```diff
@@ -106,6 +109,25 @@ def __init__(self, opt_dict):
             else:
                 raise KeyError(f"The keyword {param} is not available in self.__class__.__name__.")
 
+        copt_dict = opt_dict.copy()
```

Comment:
In this file I think you forgot to remove the previous code block, which is now replaced by your solution (just keep default_backend_options)